src/users/users.service.ts

```diff
@@ -27,4 +32,42 @@ export class UsersService {
 			message: successMessage.userCreated,
 		};
 	}
+
+	async sendEmailVerification(email: string): Promise<boolean> {
+		const user = await this.userRepository.findOne({ where: { email: email } });
```

Comment:
```suggestion
		const user = await this.userRepository.findOne({ where: { email } });
```

Reply:
Also a personal preference, you may or may not follow this.

If you're injecting `UserRepository` in `UserService`, you can simply name it `repo` instead of `usersRepository`. This leads to much cleaner code, since the module's own repository is what you'll be using primarily

```suggestion

		const user = await this.repo.findOne({ where: { email: email } });
```

Reply:
Yeah you are right but we might need to inject other repositories in the future so it might be a bit confusing with your approach. 
PS: Partham did this part of configuration.